### PR TITLE
MBS-8998: Display CPDL Score links in the sidebar

### PR DESCRIPTION
**Pull request by anshuman73**

### CODE REVIEW

File: lib/MusicBrainz/Server/Entity/URL/CPDL.pm [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
```
Comment (y-van-z) [inline lib/MusicBrainz/Server/Entity/URL/CPDL.pm]: This address is no longer current, see https://github.com/metabrainz/musicbrainz-server/pull/347#r73416211.

File: lib/MusicBrainz/Server/Entity/URL/CPDL.pm [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name {
+    my $self = shift;
+
+    return "Score at CPDL";
```
Comment (y-van-z) [inline lib/MusicBrainz/Server/Entity/URL/CPDL.pm]: Maybe that should be labelled more explicitly as "Choral Public Domain Library". @reosarevok: Any opinion?

Comment (reosarevok) [inline lib/MusicBrainz/Server/Entity/URL/CPDL.pm]: That seems way too long, and we say "Score at IMSLP"

File: lib/MusicBrainz/Server/Data/URL.pm [hunk 3]
```diff
@@ -34,6 +34,7 @@ my %URL_SPECIALIZATIONS = (
     'CDBaby'              => qr{^https?://(?:www.)?cdbaby.com/}i,
     'ChangeTip'           => qr{^https?://(?:www.)?changetip.com/tipme/}i,
     'CiNii'               => qr{^https?://(?:www.)?ci.nii.ac.jp/}i,
+    'CPDL'                => qr{^https?://(?:www.)?cdpl.org/wiki/}i,
     'Commons'             => qr{^https?://commons.wikimedia.org/wiki/File:}i,
```
Comment (anshuman73) [inline lib/MusicBrainz/Server/Data/URL.pm]: but I don't think a score link will ever need to match anything out of  http://www2.cpdl.org/wiki/ , would it ?

Comment (y-van-z) [inline lib/MusicBrainz/Server/Data/URL.pm]: No it won't, that is exactly why I crossed it out of my review two days ago. 😉 Neither will `www`. And `d` and `p` are inverted, that is, it does not match anything but misspelled URLs.

Comment (anshuman73) [inline lib/MusicBrainz/Server/Data/URL.pm]: oh, well, sorry. Missed that.

**Comment by anshuman73** [comment]
> (?:www.)? remains unneeded

Then I guess it's fair to remove that matching group.
And given that the unclean URLs are too being cleaned and the new added links will already be cleaned, it makes no sense for it to be there.

I'll remove them in a sec

**Pull request merged**